mlir/include/mlir/Transforms/Passes.td

```diff
@@ -586,4 +586,23 @@ def CompositeFixedPointPass : Pass<"composite-fixed-point-pass"> {
   ];
 }
 
+def SelectPass : Pass<"select-pass"> {
+  let summary = "Select pass";
+  let description = [{
+    Select pass allows to run multiple different set of passes based on
+    attribute value on some top-level op.
```

Comment:
Would this be on the top level it's run on?

Reply:
Technically, not necessarily on top-level op, just on this op pass is rooted on, clarified the doc.